docs/demo/new-bank-account-usecase.md

```diff
@@ -0,0 +1,80 @@
+     - The `Wallet` registers with the [DIDComm Mediator/Router](https://github.com/hyperledger/aries-framework-go/blob/master/docs/didcomm_mediator.md).
+1. Get Drivers License and Connect to Assurance Issuer: 
+   - UI Navigation 
+     - On [`Drivers License + Assurance Issuer`](#components) page, click on `Issuer Driving Licence` button. 
+     - Login to the account with default username/password and click `agree` on the consent page.
+     - Click `Next` on the window, select the `wallet` and click `Store and Connect`.    
+   - Details 
+     - The Issuer Adapter provides following details to wallet through [CHAPI](https://w3c-ccg.github.io/credential-handler-api/).
+         - Drivers License
+         - DIDComm Invitation to connect to issuer adapter
+         - Manifest Credential with details about Assurance Issuer
+     - The Wallet performs [Aries DID Exchange protocol](https://github.com/hyperledger/aries-rfcs/tree/master/features/0023-did-exchange) with the Issuer Adapter
+     - The Wallet responds with success message through CHAPI.
+1. Verify Drivers License in the Wallet:
+   - UI Navigation 
+     - Go to [`Wallet`](#components) and confirm the data in `Drivers License` under stored credentials. 
+1. Connect to Credit Report Issuer:
+   - UI Navigation 
+      - Browse to [`Credit Report Issuer`](#components) and click on `Get your credit report` button. 
+      - Login to the account with default username/password and click `agree` on the consent page.
+      - Click `Next` on the window, select the `wallet` and click `Connect`. 
+   - Details 
+     - The issuer adapter provides following details to wallet through [CHAPI](https://w3c-ccg.github.io/credential-handler-api/).
+         - DIDComm Invitation to connect to issuer adapter
+         - Manifest Credential with details about Credit Report Issuer
+1. Open a Bank Account
+   - UI Navigation 
+      - Navigate to [`Bank`](#components)'s website and click on `Open Bank Account` button.
+      - Click `Next` on the window and select the `wallet`.
+      - The next page shows the `Drivers License` along with `Drivers License Assurance` and `Credit Report` issuer.
+      - Select all 3 and click on `Share`.
+      - The user data will be displayed on the page with `success message`
+   - Details
+     - RP connects to RP adapter through OIDC protocol. 
+     - The RP adapter provides following details to wallet through [CHAPI](https://w3c-ccg.github.io/credential-handler-api/).
+         - DIDComm Invitation to connect to rp adapter
+         - [Presentation Exchange](https://identity.foundation/presentation-exchange/) request
+            - Request for Drivers License
+            - Request for Authorization to access Drivers License Assurance data
+            - Request for Authorization to access Credit Report data
+     - For Authorization, the wallet connects to the Issuer Adapter of Drivers License Assurance and Credit Report through 
```

Comment:
Can we add a note that we plan to add DIDComm blinded routing in a subsequent release.

Reply:
[Added](https://github.com/trustbloc/edge-sandbox/compare/474dd7acb5782977dbe92fe03e0f6be314fea7d8..72163290a26c71be44d74b30e6b5b9f254fb0597#diff-316073c863ac197261edb25cfb44d1d3R72) a note.